### PR TITLE
Web m2x options config parameters 70

### PR DESCRIPTION
**Pull request by njeudy**
Add ir_config_parameter compatibility for setup full system wide m2x options.

### CODE REVIEW

**Comment by njeudy** [comment]
Someone can say why test did pass ?
I load it correctly on V7 and test manually in runbot. This works too :) thanks

**Comment by pedrobaeza** [comment]
Runbot is not yet working properly, so don't worry about it.

**Comment by njeudy** [comment]
Ok, thanks, so backport to 7.0 is here :) !

**Comment by pedrobaeza** [comment]
Let me check it and I'll tell you later

File: web_m2x_options/__openerp__.py [hunk 1]
```diff
@@ -16,19 +16,19 @@
...
-``<field name="partner_id" options="{'limit': 10, 'create': false,
-'create_edit': false}"/>``
+<field name="partner_id"
+options="{'limit': 10, 'create': false, 'create_edit': false}"/>
 
 Note:
 -----
 
-If one of those options are not set, many2one field uses default many2one
+if one of those options are not set, many2one field use default many2one
```
Comment (pedrobaeza) [inline web_m2x_options/__openerp__.py:25]: As this is v7, you should also include here the changes in the description.

**Comment by pedrobaeza** [comment]
Hi, Nicolas, I have tried it, and I can get it working. I have introduced the system parameter, but nothing happens. Can you please check?

**Comment by njeudy** [comment]
Ok will try tomorrow. did you refresh js when installing ?
Dis you see web_m2x_option in js list when using debug mode ?

**Comment by pedrobaeza** [comment]
Yeah, I refresh browser, but I haven't done any kind of JS debugging.

**Comment by legalsylvain** [comment]
Hi, I tried and I have an error after installing the module.
When I use dropdown in form view (example 'res.partner' field 'title'), i get the following client error : 

´´´TypeError: x[1].split is not a function
http://localhost:8069/web/webclient/js?db=test__web_m2x_options:6359´´´

Regards.

**Comment by njeudy** [comment]
Ok I will try this and correct.

**Comment by mcassuto** [comment]
@njeudy : hello, what is the status on this ? Please tell if I can help for testings !

**Comment by mcassuto** [comment]
oups... don't misinterpret the msg "mcassuto referenced this pull request from a commit in mcassuto/CP-modules 4 hours ago"...
I did a quick fix to add the global options on the 7.0 official version on my own repo; it is not a backport of version 8.0 so I probably missed some functionalities....

File: web_m2x_options/static/src/js/form.js [hunk 2]
```diff
@@ -137,6 +167,28 @@ openerp.web_m2x_options = function (instance) {
                 new instance.web.form.M2ODialog(this).open();
             }
         },
+	
+	start: function() {
+	    this._super.apply(this, arguments);
+	    return this.get_options();
+	},
+
+	get_options: function() {
```
Comment (hbrunn) [inline web_m2x_options/static/src/js/form.js:176]: is this code duplication necessary?

File: web_m2x_options/README.rst [hunk 3]
```diff
@@ -52,12 +88,15 @@ Example
...
+           options="{'limit': 10, 'create': false, 'create_edit': false}"/>
     ...
 
 Note
 ----
 
-Double check that you have no inherited view that remote ``options`` you set on a field ! 
-If nothing work, add a debugger in the first ligne of ``get_search_result method`` and enable debug mode in OpenERP. When you write something in a many2one field, javascript debugger should pause. If not verify your installation.
-
+Double check that you have no inherited view that remote ``options`` you set
```
Comment (clonedagain) [inline web_m2x_options/README.rst:98]: "remote" should read "removes"

**Comment by JordiBForgeFlow** [comment]
I think that this PR can be closed, as https://github.com/OCA/web/pull/114 implemented the feature, right?

**Comment by OSguard** [comment]
yes already implemented

**Pull request closed without merging**